README.md

```diff
@@ -90,6 +90,18 @@ D.get('year', date); // 2015
 
 ```
 
+###unixTime
+`String -> Date -> Number | Error`
+Returns the unix time of the supplied Javascript date object in the specified
```

Comment:
UnixTime is defined as the time since epoch in seconds, so it shouldn't take a String param.
Maybe we should rename it to `convertTo` instead ? and then support minutes, hours etc?
I still like the idea of having `unixTime()` but I think it should follow the standard...